### DISCUSSION TITLE
Some nuget process filling up drive on C:\Windows\Temp\NuGet\TempCache

### DISCUSSION THREAD

**Discussion by rpaz-qontigo** [NuGet/Home#11479] [Q&A]
This started happening a while ago, and I can't trace exactly what is causing this. I've noticed drive alerts for many servers runing TeamCity Agents and Octopus Tentacle process. I can only guess this all started when using Artifactory as our nuget feed.

But the question is, how I can pinpoint who is writing files on C:\Windows\Temp\NuGet\TempCache, since all config or commands points to other directories. Where is this set?

Cleaning all local caches produces this output:
PS C:\Windows\system32> dotnet nuget locals all --clear
Clearing NuGet HTTP cache: C:\Windows\system32\config\systemprofile\AppData\Local\NuGet\v3-cache
Clearing NuGet global packages folder: C:\Windows\system32\config\systemprofile.nuget\packages
Clearing NuGet Temp cache: C:\Windows\TEMP\NuGetScratch
Clearing NuGet plugins cache: C:\Windows\system32\config\systemprofile\AppData\Local\NuGet\plugins-cache

**Comment by dominoFire** [comment]
No NuGet clients (nuget.exe, dotnet.exe, VS) uses that temp folder

Current implementation uses Users temp folder.

In case of NuGet server you may be interested in https://github.com/NuGet/NuGetGallery/issues/5230
